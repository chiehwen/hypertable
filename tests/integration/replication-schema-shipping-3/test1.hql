CREATE NAMESPACE "/rep";
CREATE NAMESPACE "/source";
USE "/source";
CREATE TABLE Repl (c) REPLICATE TO "cluster";
ALTER TABLE Repl ADD (d, e, f);
