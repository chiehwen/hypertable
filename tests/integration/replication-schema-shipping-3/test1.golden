
Welcome to the hypertable command interpreter.
For information about Hypertable, visit http://hypertable.com

Type 'help' for a list of commands, or 'help shell' for a
list of shell meta commands.

CREATE NAMESPACE "/rep";
CREATE NAMESPACE "/source";
USE "/source";
CREATE TABLE Repl (c) REPLICATE TO "cluster";
Error: MasterClient operation create_table('source/Repl') failed - HYPERTABLE induced failure
